swapping concepts landing page (#1650)
temporalio/documentation: docs/concepts.md

---
id: concepts
title: Temporal concepts
description: The following concepts have specific definitions within the context of the Temporal Platform.
sidebar_label: Concepts
tags:
  - introduction
---

The [Temporal Platform](/temporal) provides developers a suite of effective tools for building reliable applications at scale.

The concepts, components, and features of the platform are described in detail across the concept guides.

- [Temporal](/temporal)
- [Workflows](/workflows)
- [Activities](/activities)
- [Retry Policies](/retry-policies)
- [Tasks](/tasks)
- [Workers](/workers)
- [Clusters](/clusters)
- [Namespaces](/namespaces)

> - [Visibility](/visibility)

Reviewer: The sequence of terms in this list differs from the current version of the sidebar. Which one should rule?

> - [Tasks](/tasks)
> - [Workers](/workers)
> - [Clusters](/clusters)
> - [Namespaces](/namespaces)
> - [Visibility](/visibility)

Author: ```suggestion
- [Workers](/workers)
- [Tasks](/tasks)
- [Clusters](/clusters)
- [Visibility](/visibility)
- [Namespaces](/namespaces)
```

For the complete list of defined terms, see the [Glossary](/glossary).

> Information on how to implement the developer facing features is available in the [Application development guide](/application-development).

Reviewer: ```suggestion
Information on how to implement the developer-facing features is available in the [Application development guide](/application-development).
```

> Information on how to setup and operate a Temporal Cluster is available in the [Cluster deployment guide](/cluster-deployment-guide).

Reviewer: ```suggestion
Information on how to set up and operate a Temporal Cluster is available in the [Cluster deployment guide](/cluster-deployment-guide).
```
